sdk/docs/V1alpha1KFServiceStatus.md

```diff
@@ -7,7 +7,7 @@ Name | Type | Description | Notes
 **conditions** | [**list[KnativeCondition]**](KnativeCondition.md) | Conditions the latest available observations of a resource&#39;s current state. | [optional] 
 **default** | [**V1alpha1StatusConfigurationSpec**](V1alpha1StatusConfigurationSpec.md) |  | [optional] 
 **observed_generation** | **int** | ObservedGeneration is the &#39;Generation&#39; of the Service that was last processed by the controller. | [optional] 
```

Comment:
I'm confused. I don't think we surface this in our API.

Reply:
@ellis-bigelow The `KFServiceStatus` includes `duckv1beta1.Status` in the [kfservice_type.go](https://github.com/kubeflow/kfserving/blob/0ee00b3beec2ca9fb4eb6b64da1264ccb9b80914/pkg/apis/serving/v1alpha1/kfservice_types.go#L102), the `duckv1beta1.Status `includes [ObservedGeneration](https://github.com/knative/pkg/blob/2b848f71969c997809a2dbea8351eaadb23e9bc9/apis/duck/v1beta1/status_types.go#L59). So if we use the openapi-gen to generate, the `observed_generation` shown here, and in kfservice CRD define, also included [observedGeneration](https://github.com/kubeflow/kfserving/blob/0ee00b3beec2ca9fb4eb6b64da1264ccb9b80914/config/default/crds/serving_v1alpha1_kfservice.yaml#L296), as below:
```
[root@jinchi1 kfserving]# kubectl describe crd kfservices.serving.kubeflow.org |grep -i observedGeneration
              Description:  ObservedGeneration is the 'Generation' of the Service that was last processed by the controller.
```

Seems there is a problem, in the `kfservice_type.go`, we involved the `duckv1beta1.Status`, but only handled the `Conditions` and ignore `ObservedGeneration`, seems also we need to handle the `ObservedGeneration`? or update the `kfservice_type.go` to just involve `apis.Conditions`？WDYT?

Reply:
ye I think it is good to handle this, knative service has that.